recipes/ntp/ntp.inc

```diff
@@ -98,3 +106,10 @@ FILES_${PN}-ntpdate = "${sysconfdir}/network/if-up.d/ntpdate"
 # ntp originally includes tickadj. It's split off for inclusion in
 # small firmware images on platforms with wonky clocks (e.g. OpenSlug)
 RDEPENDS_${PN}-ntpd = "${PN}-tickadj ${PN}-config ${PN}-init"
```

Comment:
Just added it, it does kinda belong with the one adding support for an alternative init.